tests/phpunit/Checker/Checks/Performant_WP_Query_Params_Check_Tests.php

```diff
@@ -0,0 +1,62 @@
+<?php
+/**
+ * Tests for the Performant_WP_Query_Params_Check class.
+ *
+ * @package plugin-check
+ */
+
+use WordPress\Plugin_Check\Checker\Check_Context;
+use WordPress\Plugin_Check\Checker\Check_Result;
+use WordPress\Plugin_Check\Checker\Checks\Performant_WP_Query_Params_Check;
+use WordPress\Plugin_Check\Test_Utils\TestCase\Static_Check_UnitTestCase;
+
+class Performant_WP_Query_Params_Check_Tests extends Static_Check_UnitTestCase {
+
+	public function test_run_with_errors() {
+		$performant_query = new Performant_WP_Query_Params_Check();
+		$check_context    = new Check_Context( TESTS_PLUGIN_DIR . '/tests/phpunit/testdata/plugins/test-plugin-performant-wp-query-params-errors/load.php' );
```

Comment:
After merging `trunk` back in, please use the new constant here that was introduced in #189.